gui/window.go

```diff
@@ -603,6 +605,79 @@ func (window *Window) launchWelcomeView() {
 // launchMenuView launches the menu view
 func (window *Window) launchMenuView() {
 	window.menu.currentPage.StoreChanges()
+
+	// If syscheck fails, launch a dialog box and force the user to exit
+	if retErr := syscheck.RunSystemCheck(true); retErr != nil {
+		contentBox, err := gtk.BoxNew(gtk.ORIENTATION_HORIZONTAL, 10)
+		contentBox.SetHAlign(gtk.ALIGN_FILL)
+		contentBox.SetMarginBottom(common.TopBottomMargin)
+		if err != nil {
+			log.Warning("Error creating box")
+			return
+		}
+
+		//Custom CSS class for this dialog - make our error icon red(ish)
```

Comment:
Do we need the following code. Is it actually setting the icon to the orange color? Can we do this by just creating a corresponding style.css entry?

		//Custom CSS class for this dialog - make our error icon red(ish)
		provider, err := gtk.CssProviderNew()
		if err != nil {
			log.Warning("Error creating CSS provider")
			return
		}
		customTheme := ".dialog image {color: #FDB814;}"
		if err := provider.LoadFromData(customTheme); err != nil {
			log.Warning("Error creating CSS from internal data")
			return
		}

		screen, err := gdk.ScreenGetDefault()
		if err != nil {
			log.Warning("Error getting screen")
			return
		}
		gtk.AddProviderForScreen(screen, provider, gtk.STYLE_PROVIDER_PRIORITY_APPLICATION)

Reply:
You are correct all this code does is change the icon color for this dialog box (white -> yellow/orange)

I didn't include this in style.css because that would change the icons for all dialog boxes.
Maybe we want that? Or should we define a new type of dialog (informational vs error dialogs)?

Reply:
Yeah.. To keep it consistent with rest of the application, can we please have this as a style.css entry... Something like ".dialog-warning"

Reply:
But on a separate note, while testing I still see the icon as Red color and not Orange. Are you able to see it as Orange?

Reply:
Fixed - the color is the same as the .label-warning in the css:
`#FDB814`